packages/synapse-react-client/src/components/GenericCard/GenericCardIcon.tsx

```diff
@@ -0,0 +1,63 @@
+import IconSvg, { type2SvgIconName } from '@/components/IconSvg/IconSvg'
+import { IconProps } from '@/components/index'
+import { Icon } from '@/components/row_renderers/utils/index'
+import { ImageFileHandle } from '@/components/widgets/ImageFileHandle'
+import { FileHandleAssociation } from '@sage-bionetworks/synapse-types'
+
+export type GenericCardIconProps = {
+  /** May be used to determine the icon if `useTypeForIcon` is true */
+  type: string
+  /** If true, the icon is determined using the 'type' field passed to the generic card
+   * @default false */
+  useTypeForIcon?: boolean
+
+  imageFileHandleId?: string
+  thumbnailRequiresPadding?: boolean
+  fileHandleAssociation?: FileHandleAssociation
+
+  iconOptions?: IconProps['iconOptions']
+  iconValue?: IconProps['value']
+}
+
+export function GenericCardIcon(props: GenericCardIconProps) {
```

Comment:
This component was basically an internal component of GenericCard, is now a standalone component.